package.json

```diff
@@ -47,6 +47,6 @@
   ],
   "engines": {
     "node": ">=18.17.0",
-    "pnpm": "8.9.0"
+    "pnpm": ">=8.9.0"
```

Comment:
We could upgrade pnpm version on a separate PR, the version is fixed for easier reproduction.